test/unit/fixtures/verify/src/scss/class-name-format/valid.scss

```diff
@@ -0,0 +1,13 @@
+.hyphenated-lowercase {
+  content: '';
+}
+
+.snake_case {
+  content: '';
+}
+
+// we allow leading with an underscore,
+// but maybe shouldn't for css class names
+._with-leading-underscore {
```

Comment:
🤔 I wonder if we supported this by accident

Reply:
probably, there's quite a few things like this we might want to tidy up